internal/broker/plans_schema.go

```diff
@@ -454,14 +469,26 @@ func NewModulesSchema(rejectUnsupportedParameters bool) *Modules {
 					Description:          "Default modules",
 					AdditionalProperties: false,
 				},
+				ControlsOrder: []string{"default", "channel"},
 				Properties: ModulesDefaultProperties{
-					Type{
+					Default: Type{
 						Type:        "boolean",
-						Title:       "Use Default",
+						Title:       "Use Default Modules",
 						Description: "Check the default modules in the <a href=https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-modules?version=Cloud>default modules table</a>.",
 						Default:     true,
 						ReadOnly:    true,
 					},
+					Channel: Type{
+						Type:        "string",
+						Title:       "Default Module Channel",
+						Description: "For the default modules, specifies your preferred default release channel: Regular or Fast. For details, see <a href=https://help.sap.com/docs/btp/sap-business-technology-platform/provisioning-and-update-parameters-in-kyma-environment?locale=en-US&q=IPv#modules>Modules</a>.",
+						Enum:        ToInterfaceSlice([]string{"regular", "fast"}),
+						EnumDisplayName: map[string]string{
+							"regular": "Regular - default version",
```

Comment:
Too many "defaults"!
What does it mean here: default version. If for given landscape we have default=fast set, regular is not default IMHO.